.ci/azure-pipelines-amdgpu.yml

```diff
@@ -0,0 +1,41 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+- development
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - development
+
+jobs:
+  - job: HIP_Build_Test
+    pool: amdgpu
+    timeoutInMinutes: 90
```

Comment:
Will 90 minutes be enough?

Reply:
When I ran the tests manually, it took 65 minutes.